app.js

```diff
@@ -7,6 +7,7 @@ var bodyParser = require('body-parser');
 var session = require('express-session');
 var passport = require('passport');
 var flash = require('connect-flash');
+var winston = require('winston');
```

Comment:
This will get the global singleton instance of Winston. It is also possible to create and manage our own instances if we want multiple loggers with different settings.